projects/Allwinner/patches/linux/0052-WIP-drm-bridge-synopsys-Fix-CEC-not-working-after-power.patch

```diff
@@ -35,18 +35,25 @@ index 84cc52858ffb..3c20ef3bd3c1 100644
 -			cec_notifier_phys_addr_invalidate(hdmi->cec_notifier);
 -			mutex_unlock(&hdmi->cec_notifier_mutex);
 -		}
+-
+-		if (phy_stat & HDMI_PHY_HPD)
+-			status = connector_status_connected;
+-
+-		if (!(phy_stat & (HDMI_PHY_HPD | HDMI_PHY_RX_SENSE)))
+-			status = connector_status_disconnected;
```

Comment:
I don't think removing newly backported fixes and keeping status quo is right approach. This should be actually tested.

Reply:
Traveling next couple of days. Also in Rk-drm-1000 needs the same fix / drop of patch. Didn’t get time this morning.

Reply:
5.10.150 has been released. 

The patch that I reverted is : https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c?h=v5.10.150&id=b70f8abc1a440be6802f27631ecb9acb0a158c66 to allow the existing revert patch to 

@jernejsk / @knaerzche - plan?

```
Subject: [PATCH] WIP: drm/bridge: synopsys: Fix CEC not working after power-cyclying

This fixes standby -> power-on on Rockchip platform for, at least,
RK3288/RK3328/RK3399 where CEC wasn't working after powering on again.
It might differ for other phy implementations:
The whole HPD-detection part shoud be reworked and we should in general
avoid to rely in RX_SENSE phy status (at least for HDMI), since it differs
depending on sink's implementation.
```

Reply:
@jernejsk / @knaerzche - patches for both Rockchip and Allwinner reworked to include the upstream fix.